brianlong/www.validators.app
# 188084432-add-validator-ip-to-api

#### What's this PR do?
- as in title

#### How should this be manually tested?
- check validators index and details endpoint (should contain IP)
- check the docs
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/188084432)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
